packages/components/src/TokenTutorial/TokenTutorial.tsx

```diff
@@ -28,13 +28,26 @@ export class TokenTutorial extends React.Component<TokenTutorialProps, TokenTuto
 
           const quizPayload = loading || error ? {} : data.currentUser.quizPayload;
 
-          if (this.state.isQuizStarted) {
-            return <TokenTutorialLanding handleClose={this.props.handleClose} quizPayload={quizPayload} />;
+          if (this.isQuizStarted(quizPayload)) {
+            return (
+              <TokenTutorialLanding
+                handleClose={this.props.handleClose}
+                quizPayload={quizPayload}
+                isQuizStarted={true}
+              />
+            );
           }
 
           return <TutorialWelcomeScreens handleClose={this.props.handleClose} quizPayload={quizPayload} />;
         }}
       </Query>
     );
   }
+
+  private isQuizStarted = (quizPayload: any) => {
+    if (Object.keys(quizPayload).length > 0) {
```

Comment:
Just curious, why not just return `Object.keys(...)`?

Reply:
👍 I started out doing something else and didn't notice that